src/main/java/frc/robot/AutoCommands.java

```diff
@@ -0,0 +1,124 @@
+package frc.robot;
+
+import com.pathplanner.lib.auto.NamedCommands;
+import com.pathplanner.lib.commands.PathPlannerAuto;
+
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.subsystems.CoralManipulator;
+import frc.robot.subsystems.Elevator;
+import frc.robot.subsystems.ElevatorState;
+import frc.robot.subsystems.ManipulatorState;
+import frc.robot.subsystems.Swerve;
+
+/* Notes for Auto Maker:
+ * Start at B1/B2/B3 position with preloaded game piece
+ * Add event marker "prepareL3" as robot starts moving
+ * Once in scoring position, add event marker "scoreManipulator"
+ * ONE PIECE END => End with "resetToIdle." Continue reading for TWO PIECE.
+ * At coral station, add event marker "runManipulator"
+ * Include a wait time of ~0.5-1.0 seconds to ensure intake
+ * Once coral is acquired (beambreak triggered), path to scoring position
+ * Add event marker "prepareL3" during return movement
+ * Add event marker "scoreManipulator" when in position
+ * End with "resetToIdle"
+ */
+
+public class AutoCommands {
+    private SendableChooser<Command> chooser;
+
+    public AutoCommands(Swerve swerve, Elevator elevator, CoralManipulator manipulator) {
+
+        // scoring commands for different levels: moves elevator to right position for scoring
+        NamedCommands.registerCommand("prepareL2", 
+            elevator.changeState(ElevatorState.L2)
```

Comment:
I'd advise using the state machine even for these unless you have a reason not to